src/plugins/manualRowResize/manualRowResize.js

```diff
@@ -324,6 +324,23 @@ class ManualRowResize extends BasePlugin {
     return null;
   }
 
+  /**
+   * Returns the actual height for the provided row index.
+   *
+   * @param {number} row Visual row index.
+   * @returns {number} Actual row height.
+   */
+  getActualRowHeight(row) {
```

Comment:
Have you thought about marking the method as private (private for docs)? Is it supposed to be public?

Reply:
Good idea 👍